src/main/java/com/redhat/red/build/koji/model/converter/KojiBuildTypeQueryConverter.java

```diff
@@ -1,100 +0,0 @@
-/**
```

Comment:
This converter never worked, so I have removed it. It was supposed to create a parameter `query` with the map, not a map as a direct argument. I created a new class `KojiBuildTypeQueryQuery` (yes, not the best name) which is a normal `@StructPart`. We keep the fields in the `KojiBuildTypeQuery` class which are really fields in the `query` to match the other queries. The API itself is inconsistent on the Koji side. This is now tested by `ExternalKojiQueryOptsTest` as working and I have expanded the unit test `ListBuildTypesRequestTest` to test with a query. Previously, it had just tested without, so we never caught that it did not work.